lib/erlef_web/live/academic_papers_live.ex

```diff
@@ -0,0 +1,24 @@
+defmodule ErlefWeb.AcademicPapersLive do
```

Comment:
Look ma! No javascript! 🚀

Reply:
Oh there's javascript. We just don't have to implement it :)